remote/src/main/java/Main.java

```diff
@@ -0,0 +1,18 @@
+public class Main {
+    public static void main(String[] args) throws InterruptedException {
+        RobotController controller = new RobotController();
+
+        controller.startController("192.168.1.12:50051");
```

Comment:
I still think that the IP of the robot should be taken as a dynamic input, as originally intended.

```suggestion
        if (args.length < 1) {
            throw new MissingArgumentException("Please provide an IP and port number (e.g 192.168.0.97:50051)");
        }
        RobotController controller = new RobotController();

        controller.startController(args[0]);
        
```

Reply:
I implemented it in the newest commit